src/main/java/chess/Board.java

```diff
@@ -0,0 +1,87 @@
+package chess;
+
+import chess.pieces.Pawn;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Board {
+    private static final int SIZE = 8;
+    private static final int WHITE_PAWNS_LINE = 1;
+    private static final int BLACK_PAWNS_LINE = 6;
+
+    private final List<List<Pawn>> pawns;
+
+    public Board() {
+        this.pawns = new ArrayList<>();
+    }
+
+    public void initialize() {
+        for (int line=0; line<SIZE; line++) {
```

Comment:
```suggestion
        for (int line = 0; line < SIZE; line++) {
```

코드 포맷은 항상 신경써 주세요. 자동 코드 포맷을 활용하는 것도 좋습니다.